src/AutoPilotPlugins/APM/APMAutoPilotPlugin.cc

```diff
@@ -128,7 +128,7 @@ const QVariantList& APMAutoPilotPlugin::vehicleComponents(void)
             _tuningComponent->setupTriggerSignals();
             _components.append(QVariant::fromValue((VehicleComponent*)_tuningComponent));
 
-            if(_vehicle->parameterManager()->parameterExists(MAV_COMP_ID_UDP_BRIDGE, "MNT_RC_IN_PAN")) {
+            if(_vehicle->sub() || _vehicle->parameterManager()->parameterExists(MAV_COMP_ID_UDP_BRIDGE, "MNT_RC_IN_PAN")) {
```

Comment:
Oops sorry. Doesn't this still pop missing parameter errors though because of the summary page which also references the MNT* params. Also noticing that my cut/paste is screwed up using `MAV_COMP_ID_UDP_BRIDGE`.

Reply:
Check the last version of the patch, it should fix it as original intended.

Reply:
Thanks @patrickelectric